dockerfiles/che/entrypoint.sh

```diff
@@ -246,6 +244,7 @@ init() {
     export CHE_DOCKER_IP_EXTERNAL=${HOSTNAME}
   fi
   ### Necessary to allow the container to write projects to the folder
+  export CHE_WORKSPACE_STORAGE__MASTER__PATH=/data/workspaces
```

Comment:
why do we need to duplicate the property CHE_WORKSPACE_STORAGE ? 
so we have
 CHE_WORKSPACE_STORAGE__MASTER__PATH=/data/workspaces
and
 CHE_WORKSPACE_STORAGE="${CHE_DATA_HOST}/workspaces"

also /data should be replaced by ${CHE_DATA} property

Reply:
Because these are different paths?